target/linux/ath79/dts/qca9563_glinet_gl-ar750s.dtsi

```diff
@@ -44,19 +46,19 @@
 	leds {
 		compatible = "gpio-leds";
 
-		power: power {
+		led_power: power {
 			label = "gl-ar750s:green:power";
 			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
 			default-state = "keep";
 		};
 
-		wlan2g {
+		led_wlan2g: wlan2g {
```

Comment:
Do you need those labels (wlan2g/wlan5g) for anything?

Reply:
Good coding practice, consistency.

_Node_ labels in a DTS (in contrast to the `label` property) do not appear in the DTB, so do not add to the kernel size.

Reply:
So in the end it's a matter of taste again. I'd have preferred to not set things which are "useless" (no offense) for my own stuff. Anyway, keep them if you prefer it this way.